src/HVGroup.cpp

```diff
@@ -7,51 +10,94 @@ HVGroup::HVGroup(Interface& m_interface):
 
         m_box = new QGroupBox("HV Settings", this);
 
-        m_label = new QLabel("HV value:");
-        
-        m_value_label = new QLabel("0");
-        m_value_label->hide();
+        m_layout = new QGridLayout();
 
-        m_spin = new QSpinBox(0);
-        m_spin->setRange(0, 2000);
-        m_spin->setSingleStep(1);
+        int temp_iterator = 0;
+        for (int setHVValue : m_interface.getConditions().getHVPMTSetValues()){
```

Comment:
I think I'll keep it that way, at least for this one, because I need the temp_iterator later. I would then anyway have to define an int before the loop and set it to the "id" value.